cli_tools/common/utils/logging/logger.go

```diff
@@ -33,22 +34,31 @@ func (e *LogEntry) String() string {
 	return fmt.Sprintf("%s %s", e.LocalTimestamp.Format("2006-01-02T15:04:05Z"), e.Message)
 }
 
-// Logger is responsible for logging to stdout
-type Logger struct {
-	Prefix string
-}
-
 // LoggerInterface is logger abstraction
 type LoggerInterface interface {
 	Log(message string)
 }
 
-// NewLogger creates a new logger which uses prefix for all messages logged
-func NewLogger(prefix string) *Logger {
-	return &Logger{Prefix: prefix}
+// NewStdoutLogger creates a new logger which uses prefix for all messages logged.
+// All messages are sent to stdout.
+func NewStdoutLogger(prefix string) LoggerInterface {
+	return stdoutLogger{prefix: prefix}
 }
 
+type stdoutLogger struct{ prefix string }
```

Comment:
This could be made more flexible by parametrizing the logger with a file to write to. Stdout is just a file in GoLang so default implementation will use os.Stdout

Reply:
My intuition is that this is beyond the scope of this review -- this PR just renames the method to make it clear what it was already doing.